src/store/cesium.ts

```diff
@@ -20,9 +23,11 @@ Cesium.RequestScheduler.maximumRequestsPerServer = 3;
 
 export const cesiumViewer = ref();
 
+export const useMap = ref(false);
+
 export const tileImageParams: Record<string, TileParamsType> = {};
 
-const footprintEntities: Record<string, Entity> = {}; // Cesium.Entity
+const footprintEntities: Record<string, Array<Entity>> = {}; // Cesium.Entity
```

Comment:
Is `Array<Entity>` equivalent to `Entity[]`? If so, let's just consistently use the latter notation.

Reply:
Changed in 73d01e4